hieradata/cluster/konkong.yaml

```diff
@@ -0,0 +1,126 @@
+---
+clustershell::groupmembers:
+  konkong: {group: "konkong", member: "konkong[01-03]"}
+cni::plugins::checksum: "f3a841324845ca6bf0d4091b4fc7f97e18a623172158b72fc3fdcdb9d42d2d37"
+cni::plugins::enable: ["macvlan"]
+cni::plugins::version: "1.2.0"
+profile::core::common::manage_powertop: true
+profile::core::ospl::enable_rundir: true
+profile::core::rke::enable_dhcp: true
+profile::core::rke::version: "1.3.12"
+profile::nm::connections:
+  eno1np0:
+    content: |
+      [connection]
+      id=eno1np0
+      uuid=f330f829-20cc-b829-67b0-18086a5fe6fa
+      type=ethernet
+      autoconnect=false
+      interface-name=eno1np0
+
+      [ethernet]
+
+      [ipv4]
+      method=disabled
+
+      [ipv6]
+      method=disabled
+  eno2np1:
+    content: |
+      [connection]
+      id=eno2np1
+      uuid=de9904c8-9577-1a17-36b1-34b94132f06a
+      type=ethernet
+      autoconnect=false
+      interface-name=eno2np1
+
+      [ethernet]
+
+      [ipv4]
+      method=disabled
+
+      [ipv6]
+      method=disabled
+  enp4s0f3u2u2c2:
+    content: |
+      [connection]
+      id=enp4s0f3u2u2c2
+      uuid=283f3035-13d7-4c87-9d7a-7d47861fa1f9
+      type=ethernet
+      autoconnect=false
+      interface-name=enp4s0f3u2u2c2
+
+      [ethernet]
+
+      [ipv4]
+      method=disabled
+
+      [ipv6]
+      method=disabled
+  enp129s0f0:
+    content: |
+      [connection]
+      id=enp129s0f0
+      uuid=688bf5bf-d649-34b4-15eb-b07c50ac43f8
+      type=ethernet
+      interface-name=enp129s0f0
+
+      [ethernet]
+
+      [ipv4]
+      method=auto
+
+      [ipv6]
+      method=disabled
+  enp129s0f1:
+    content: |
+      [connection]
+      id=enp129s0f1
+      uuid=46d19ce1-bcab-7e77-6fc7-b730b26c54b1
+      type=ethernet
+      autoconnect=false
+      interface-name=enp129s0f1
+
+      [ethernet]
+
+      [ipv4]
+      method=disabled
+
+      [ipv6]
+      method=disabled
+  enp129s0f1.2502:
+    content: |
+      [connection]
+      id=enp129s0f1.2502
+      uuid=e74daaad-4d73-4015-bb26-83c460ddf572
+      type=vlan
+      interface-name=enp129s0f1.2502
+      master=br2502
+      slave-type=bridge
+
+      [ethernet]
+
+      [vlan]
+      flags=1
+      id=2502
+      parent=enp129s0f1
+
+      [bridge-port]
+  br2502:
+    content: |
+      [connection]
+      id=br2502
+      uuid=7dd05d98-a9c3-4569-a7c3-00316b13c0eb
+      type=bridge
+      interface-name=br2502
+
+      [ethernet]
+
+      [bridge]
+      stp=false
+
+      [ipv4]
+      method=disabled
+
+      [ipv6]
+      method=disabled
```

Comment:
I would suggest to move the config into site/ls/cluster/konkong.yaml

Reply:
i just use the same hierachy as Manke, if this is wrong, we should change them both. but i think its OK, all the other clusters have the same config here.
<img width="186" alt="imagen" src="https://user-images.githubusercontent.com/1090349/224422153-97db25fa-29f1-45bf-910c-473f38256776.png">

Reply:
yeap, you're right; highly unlikely we'll have another cluster with the same name